<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xdiskguiClass</class>
 <widget class="QWidget" name="xdiskguiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xdiskgui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 254, 255);</string>
  </property>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="filelistWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>大小</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="portBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>8888</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务器IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>146</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务器端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dir">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>16</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>读取路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dirEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>./files</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xdiskgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>xdiskguiClass</receiver>
   <slot>Refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadButton</sender>
   <signal>clicked()</signal>
   <receiver>xdiskguiClass</receiver>
   <slot>Upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>xdiskguiClass</receiver>
   <slot>Download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Refresh()</slot>
  <slot>Upload()</slot>
  <slot>Download()</slot>
 </slots>
</ui>
